docs/source/concept_models/vivarium_iv_iron/concept_model.rst

```diff
@@ -286,6 +286,12 @@ For model versions 2+:
 
 * :ref:`Maternal Body Mass Index <2019_risk_exposure_maternal_bmi>`
 
+.. todo::
+
+  Ali to update this risk exposure to be specific to BMI < 18.5 using the GBD estimates of continuous BMI exposure using modelable entity IDs 2548 and 2548. 
```

Comment:
Did you mean to have 1 MEID here or 2? "modelable entity IDs 2548 and 2548"

Reply:
Thanks for catching! Updated to reflect the correct MEIDs.